src/js/node/worker_threads.ts

```diff
@@ -224,6 +224,10 @@ class Worker extends EventEmitter {
     this.#worker.addEventListener("open", this.#onOpen.bind(this));
   }
 
+  threadId() {
+    return _threadId;
```

Comment:
Isn't this not the `Worker`'s threadId but the parent's? I think you want `this.#worker.threadId`

Reply:
Indeed, and it must be a proper getter.